handlers/auth.go

```diff
@@ -89,6 +101,17 @@ type AuthConfig struct {
 	RoleCategories []*gaia.UserRoleCategory
 }
 
+func getPoliciesFromClaims(policyClaims interface{}) []string {
+	policies := []string{}
+	if policyClaims == nil {
+		return policies
+	}
+	for _, p := range policyClaims.([]interface{}) {
```

Comment:
Do a type assert here on `[]interface{}` to make sure it doesn't throw a type exception.